scripts/bump-version.mjs

```diff
@@ -163,6 +163,19 @@ if (prerelease) {
   nextVersion = result.nextVersion
 }
 
+if (packageFolder === 'cli') {
```

Comment:
@eunjae-lee, could we have an `enum` for this checking here?

```
const PACKAGE_FOLDER_TYPES = {
    FIELD_PLUGIN: 'field-plugin',
    CLI: 'cli',
}

...

if (packageFolder === PACKAGE_FOLDER_TYPES.CLI) {}
```

and then we could also update the `PACKAGE_FOLDERS` for make use of it...
something like that:

```
const PACKAGE_FOLDERS = [
  {
    title: 'Library',
    value: PACKAGE_FOLDER_TYPES.FIELD_PLUGIN,
  },
  {
    title: 'CLI',
    value: PACKAGE_FOLDER_TYPES.CLI,
  },
]
```

What do you think?

Reply:
hey @demetriusfeijoo based on your feedback, I just converted this to typescript!

Reply:
Thanks @eunjae-lee. 

Nice job changing it.

I was checking it locally and now it seems the ESLint is not working anymore for this script:

![image](https://github.com/storyblok/field-plugin/assets/1240591/5982ad2f-61bb-4af5-b8f5-a8982ae0f0c3)

Reply:
fixed! https://github.com/storyblok/field-plugin/pull/265/commits/421de3ffba8e48d1dd54dcc00188bee7edce6090